Sources/App/Icon/IconVariable.swift

```diff
@@ -359,6 +365,10 @@ enum IconSurfaceVariable: String, CaseIterable, GenericVariableMixable {
             return .linear
         case .cloud_cover_high:
             return .linear
+        case .convective_cloud_top:
+            return .linear
+        case .convective_cloud_base:
+            return .linear
```

Comment:
Can you set `.hermite(bounds: 0...10e9)`? This is the same I am using for DMI cloud heights